_pages/teaching.markdown

```diff
@@ -10,7 +10,49 @@ toc_sticky: true
 
 
 header:
-  image: /assets/images/wip.jpg
-  caption: "Photo credit: [**Gaelle Marcel**](https://unsplash.com/@gaellemarcel?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+  image: /assets/images/teaching/teaching.jpg
+  caption: "Photo credit: [**Diego PH**](https://unsplash.com/@jdiegoph?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
 
 ---
+
+KBSS group main teaching scope is in Semantic Web, Ontologies and Enterprise architectures in Java. We also participate in teaching database systems.
+
+Ontologies, semantic web and enterprise architectures are also in the center of topics for theses in our group.
+
+## Subjects
+
+In the winter semester 2022/23 we teach following subjects:
+
+###  Ontologies and Semantic Web (b4m36osw)
+
+The course guides students through current trends and technologies in the semantic web field.
+
+[Subject description from the faculty web](https://fel.cvut.cz/en/education/bk/predmety/58/33/p5833706.html){: .btn .btn--info}
+[Details for students at CourseWare](https://cw.fel.cvut.cz/b221/courses/b4m36osw/start){: .btn .btn--info}
```

Comment:
Also if there is a link to the newest version of those pages it would be great ... I mean if there exists a page that redirects to the newest version of the subject  (maybe even a little javascript would do that :).

Reply:
I've solved it in liquid :)

Reply:
:+1: